website/pages/docs/guide.mdx

```diff
@@ -38,22 +35,6 @@ Million.js assumes that you're already familiar with and you're using React. If
 - The limitations of blocks
 </Callout>
 
-## What's a block?
-
-Million.js is a library that enables you to create _blocks_. A block is a special [Higher Order Component (HOC)](https://legacy.reactjs.org/docs/higher-order-components.html) that can be used as a React component but is hyper-optimized for rendering speed.
-
-Blocks are essentially components wrapped by `block(){:jsx}`.
-
-```jsx {3, 7}
-import { block } from "million/react";
-
-const LionBlock = block(function Lion() {
-  return (
-    <img src="https://million.dev/lion.svg" />
-  );
-})
-```
-
 Blocks can be used just like a normal React component:
 
 ```jsx {5}
```

Comment:
In thinking about this page I'm not sure how introducing a basic example here helps. Maybe once you start getting into describing what is happening after creating a block, bringing in this info can be helpful.

For example, with this being a tutorial you want to introduce concepts as you go. Remember that Diataxis offers the suggestion that a Tutorial's purpose is to give the user basic competency. This includes throwing in a bit of info here and there that can help comprehension. 

If you want this to be a guide, however, you're taking the user through a series of instructions on how to achieve a final outcome (eg. how to create a block out of an existing application). You don't really need to add any further information than is necessary.

Reply:
I might suggest rethinking what you want out of this page (tutorial vs guide) and restructuring it based on that 🙂

Reply:
@LadyBluenotes  this is the one that prompted me to ask if we could have a meeting or just an async conversation around everything that needs major restructuring

Reply:
Got it!

I'll make a separate comment regarding all the questions.